app/views/api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+## 5th February
+
+The following experimental/sandbox endpoint has been updated:
+
+- `/test-data/generate` now accepts an optional `for_ratified_courses` query param. If this parameter is supplied and set to a non-empty string, applications will be generated for courses the organisation awards, not runs. This means subsequent calls to `/test-data/clear` will NOT delete these applications.
```

Comment:
Is the word "awards" used elsewhere to describe courses that an org ratifies? For consistency, maybe using the word "ratifies" is better?

The last point is interesting. This sort of leaves them with no way to remove applications that they generated themselves. Not sure what the best way to get around that would be

Reply:
Adding more text to explain the use-case better (HEIs ratifying School Direct courses).